cmake/CMakeLists.txt

```diff
@@ -1285,6 +1285,10 @@ if(onnxruntime_USE_OPENVINO)
 
   add_definitions(-DUSE_OPENVINO=1)
 
+  if (onnxruntime_DISABLE_GRAPH_PARTITION)
```

Comment:
do we need a build time option if there is a runtime option?
besides, the name of the option implies disabling partitioning for all providers rather than OV specific.

Reply:
we were wondering it would be nice to have a build time option as well. since some of the developers would love to experiment their models and workloads only if it's supported fully on the accelerated EP else they would like to default it back to default CPU to not run into performance issues due to sub graph partitioning.

I put the option as --disable_graph_partition which is more generic and is not specific to OpenVINO-EP because I thought if any other EP's also wanna make use of this feature in their EP's, we all can have a common build time and run time option. All they have to do is add the if() checks in their code base to reject the graph partitioning.

If you think this option isn't scalable to other EP's, I can rename it to openvino specific name like
 **--ov_disable_graph_partition**

Reply:
If it's ok to have this option common to all EP's. I would have to make some changes in the print statements and some changes in build.py file.

If you want me to just make it to specific for OpenVINO-EP for now, I will just have to rename the config option.

Let me know your thoughts george @jywu-msft , I will push a commit accordingly.